src/hotspot/share/opto/vectorIntrinsics.cpp

```diff
@@ -542,6 +584,8 @@ bool LibraryCallKit::inline_vector_nary_operation(int n) {
       }
       return false;
      }
+  } else if (VectorNode::is_bitshuffle_opcode(sopc) && !Matcher::match_rule_supported_vector(sopc, num_elem, elem_bt)) {
+    operation = gen_bitshuffle_operation(opc, elem_bt, num_elem, opd1, opd2);
   } else {
```

Comment:
I'm afraid that you should add the `Op_CompressBitsV` and `Op_ExpandBitsV` node creation in `VectorNode::make()`. If the backend support these two ops, the jvm crashes due to missing creation of these two nodes. I met the issue when I try to support the backend for ARM SVE:
```
Internal Error (panama-vector/src/hotspot/share/opto/vectornode.cpp:657), pid=152820, tid=152852
#  fatal error: Missed vector creation for 'CompressBitsV'
```